mlos_bench/mlos_bench/schedulers/base_scheduler.py

```diff
@@ -242,7 +242,6 @@ def __exit__(
         self._in_context = False
         return False  # Do not suppress exceptions
 
-    @abstractmethod
     def start(self) -> None:
         """Start the scheduling loop."""
```

Comment:
Maybe expand on this documentation so that it notes what the other functions it calls and in what sequence so someone reading the docs could know how overrides might interact.

Reply:
yep. we can do it in the next PR